<launch>
  <arg name="robot" default="scitos"/>
  <arg name="input_file_dir" default="$(find viper)/data/$(arg robot)/bothAB"/>

  <node name="generate_bar_chart" pkg="viper" type="gen_bar_chart.py" output="screen">
    <param name="input_file_dir" type="string" value="$(arg input_file_dir)"/>
  </node>
</launch>